.github/workflows/test.yml

```
@@ -31,13 +31,12 @@ jobs:
       - name: Scaffold Python package
         run: |
           pip install --no-input cruft
-          cruft create --no-input --extra-context '{"package_name": "My Package", "python_version": "3.8", "with_fastapi_api": "1", "with_typer_cli": "1"}' ./template/
+          cruft create --no-input --extra-context '{"package_name": "My Package", "python_version": "3.8", "processing_unit": "GPU", "with_fastapi_api": "1", "with_typer_cli": "1"}' ./template/
```

```suggestion
          cruft create --no-input --extra-context '{"package_name": "My Package", "python_version": "3.8", "docker_image": "radixai/python-gpu:cuda11.8-python$PYTHON_VERSION", "with_fastapi_api": "1", "with_typer_cli": "1"}' ./template/
```

For the Docker image tag, I was thinking it might be nice to follow the official Python tag structure, which looks like `python:3.8-slim`. In our case, the GPU-supported version could then be `radixai/python-gpu:3.8-cuda11.8`?